### PR TITLE
Steve

### PR DESCRIPTION
**Pull request by TheSteeZ**
added html, css, and images

### CODE REVIEW

**Review by achower — COMMENTED** [review]
on css/steve.css you styled body twice, you can combine these

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (nw0428) [inline .gitignore]: Delete this file

File: index.html [hunk 2]
```diff
@@ -8,7 +8,11 @@
   <div class="center">
     <ul>
     	<li><a href="html/david.html">David Zabner</a></li>
+<<<<<<< HEAD
```
Comment (nw0428) [inline index.html]: This still has merge conflicts in it.

**Review by nw0428 — CHANGES_REQUESTED** [review]
LGTM

**Review by nw0428 — APPROVED** [review]

**Pull request merged**